### PR TITLE
test: Add tests for register method

### LINKED ISSUE #21: feat: register (account_register)
1. Proxy the `Credentials` class received in your method to the `/register` of the [authentication-go](https://github.com/hawks-atlanta/authentication-go/blob/main/docs/spec.openapi.yaml)
2. (deprecated) Check the response and `throw` in case of any error
3. In case of success `Unmarshal` the received `JSON` and map it to the `Authorization` class for the return value

4. (new) Comply with the Open Api spec.

### PR DESCRIPTION
**Pull request by SilviaPabon**
@Woynert made the updates for register method previosly, so I just added the tests.
This approved should be close #21

### CODE REVIEW

**Comment by shoriwe** [comment]
How the progress of this has evolved?

**Comment by SilviaPabon** [comment]
> How the progress of this has evolved? @SilviaPabon

Hi @shoriwe I will mark as ready this PR and another one today.

**Comment by SilviaPabon** [comment]
@shoriwe Thinking better 💡... I will analyze something about this PR because perhaps I can unite some things about the register with login making something similar to tests that @Woynert implemented in a previous PR, like steps. So, tomorrow I hope that I will concrete this PR 😬

**Comment by SilviaPabon** [comment]
@Woynert Now this is ready for checking, thanks!

**Review by Woynert — CHANGES_REQUESTED** [review]
Return 500 on catched error.

File: app/src/test/java/gateway/ITServiceAuth.java [hunk 1]
```diff
@@ -17,6 +17,25 @@ class ITServiceAuth
 {
 	@BeforeEach void setup () { Config.initializeFromEnv (); }
 
+	@Test void Register ()
+	{
+
+		Credentials credentials = new Credentials (UUID.randomUUID ().toString (), "pass");
+
+		Credentials credentialsCopy = credentials;
```
Comment (Woynert) [inline app/src/test/java/gateway/ITServiceAuth.java:25]: This is not a copy, but a reference.

**Review by Woynert — CHANGES_REQUESTED** [review]

**Comment by PedroChaparro** [comment]
@SilviaPabon, there is a conflict :eyes:

**Review by Woynert — APPROVED** [review]
:octocat:

**Pull request merged**